themes/schema.json

```diff
@@ -1681,7 +1682,7 @@
                 "properties": {
                   "playing_icon": {
                     "type": "string",
-                    "title": "User Info Separator",
+                    "title": "User Info Separator",			
```

Comment:
This still has some whitespace and shouldn't show in the diff here

Reply:
Fixed